src/UndockedRegFreeWinRT/Nuget/readme.txt

```diff
@@ -47,6 +47,24 @@ Your application manifest should have the same name as, and be side by side to y
 Starting with Windows 10 19h1, the operating system has this feature by default and will use this information directly. Be sure to  target your application for windows so that the package automatically disables itself when this feature is available by default.
 https://docs.microsoft.com/en-us/windows/win32/sysinfo/targeting-your-application-at-windows-8-1
 
+
+Windows 8.1 Support (10/16/2020)
+Undocked egFreeWinRT can now activate types down to Windows 8.1. However, there are some minor things to account for on Windows 8.1. 
+
+1) On Windows 8.1. you may encounter 0x8007109A (This operation is only valid in the context of an app container) error when activating your type. This is caused by the app container check that exists in windows 8.1 but not on later versions.
```

Comment:
Does "[...] you may encounter error 0x8007109A (This operation...)" sound better?

Reply:
Yea kinda!